src/Utilities/results.jl

```diff
@@ -291,11 +291,14 @@ function set_dot(x::Vector, y::Vector,
     return triangle_dot(x, y, set.side_dimension, 0)
 end
 
-function set_dot(x::Vector, y::Vector, set::Union{MOI.LogDetConeTriangle,
-                                                  MOI.RootDetConeTriangle})
+function set_dot(x::Vector, y::Vector, set::MOI.RootDetConeTriangle)
     return x[1] * y[1] + triangle_dot(x, y, set.side_dimension, 1)
 end
 
+function set_dot(x::Vector, y::Vector, set::MOI.LogDetConeTriangle)
+    return x[1] * y[1] + triangle_dot(x, y, set.side_dimension, 2)
```

Comment:
Why isn't there `x[2] + y[2]` in the sum ? This is the generalization of the ExponentialCone so the scalar product should match the exponential cone one for 1x1 matrices

Reply:
Correct, I'll fix